### PR TITLE
[Refactor] package option

### PR DESCRIPTION
**Pull request by IHEII**
### Task Description

Move all options into the package option for a better user experience.

### Solution Description

Move all options into the package option for a better user experience.

### Passed Regressions

Pass all tests.

### Upgrade Compatibility

client.Withxxx -> option.Withxxx

### CODE REVIEW

File: client/option/operation_option.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ * #L%
+ */
+
+package option
+
+import (
+	"github.com/oceanbase/obkv-table-client-go/client/filter"
+)
+
+type ObkvOperationOption interface {
```
Comment (WeiXinChan) [inline client/option/operation_option.go:24]: maybe ObOperationOption is enough

File: client/option/query_option.go [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ * See the Mulan PSL v2 for more details.
+ * #L%
+ */
+
+package option
+
+import "github.com/oceanbase/obkv-table-client-go/table"
+
+type ObkvQueryOption interface {
```
Comment (WeiXinChan) [inline client/option/query_option.go:22]: ObQueryOption maybe is enough

**Comment by namewxt1220** [comment]
LGTM

File: client/option/query_option.go (hunk 2 above)
Comment (WeiXinChan) [reply client/option/query_option.go:22]: all option related should change obkv to ob

Comment (IHEII) [reply client/option/query_option.go]: Fixed.

File: client/option/operation_option.go (hunk 1 above)
Comment (IHEII) [reply client/option/operation_option.go]: Fixed.

**Pull request merged**